container-run/pkg/containerrun/containerrun_test.go

```diff
@@ -651,6 +650,23 @@ var _ = Describe("ContainerRunner", func() {
 			Expect(p).To(BeNil())
 		})
 
+		It("fails when the command exits 1", func() {
+			cr := NewContainerRunner()
+			cmd := Command{
+				Name: "bash",
+				Arg:  []string{"-c", "exit 1"},
```

Comment:
Any reason to not do this like `succeeds when…` but with an unsucessfull `ProcessState` ?

Or at least do it by setting the command to `/bin/false` or something… (I assume none of this is expected to work on Windows).

Reply:
They are in different contexts. Here we are testing `Run`, not `Wait`. The scope here is broader. As for `/bin/false` or `exit 1`, they don't really make any practical difference, right?